src/lib/gravity/index.ts

```diff
@@ -0,0 +1,25 @@
+import fetch from "node-fetch"
+
+class Gravity {
+  static HOSTS = {
+    staging: "stagingapi.artsy.net",
+    production: "api.artsy.net",
+  }
+
+  async get(endpoint: string) {
+    const token: string = process.env.TOKEN! // temp until our auth/token plumbing is hooked up
+
+    const gravityUrl: string = this.url(endpoint)
+    const headers = { "X-Access-Token": token }
+    const response = await fetch(gravityUrl, { headers })
+
+    return response
```

Comment:
Returns the response so that consumers can e.g. munge out the http status code if need be.